usr/share/rear/layout/save/FDRUPSTREAM/Linux-s390/990_copy_disklayout_file.sh

```diff
@@ -23,7 +23,17 @@ opath=$( output_path $scheme $path )
 
 if [[ "$ZVM_NAMING" == "Y" && "$ARCH" == "Linux-s390" ]] ; then 
       VM_UID=$(vmcp q userid |awk '{ print $1 }')
-      LogPrint "s390 dislayout.conf will be saved as $(opath)/$VM_UID.disklayout.conf"
-      cp $v $DISKLAYOUT_FILE ${opath}/"$VM_UID".disklayout.conf || Error "Failed to copy disklayout.conf ($DISKLAYOUT_FILE) to ${opath}/"$VM_UID".disklayout.conf"
+      if [[ -z $VM_UID ]] ; then
+            LogPrint "VM UID is not set, VM UID is set from call to vmcp.  Please make sure vmcp is available and 'vmcp q userid' returns the vm login id"
+            LogPrint "VM UID for disklayout.conf will be set to 'DEFAULT'"
+            VM_UID="DEFAULT"
+      fi
+      if [[ -z $opath ]] ; then
+            LogPrint "Warning: output path is not set, please check OUTPUT_URL in local.conf."
```

Comment:
Scripts that are run during "rear mkrescue/mkbackup"
(e.g. scripts in the `layout/save` stage - cf. the `rear -s mkrescue` output)
are normally run non-verbose so `LogPrint` messages are normally
not shown to the user while "rear mkrescue/mkbackup" runs.

Therefore `LogPrint "Warning..."` messages are normally
not shown to the user while "rear mkrescue/mkbackup" runs.

To show such messages in any case to the user use `LogPrintError "..."`
cf. the `LogPrintError` function comment in lib/_input-output-functions.sh
https://github.com/rear/rear/blob/master/usr/share/rear/lib/_input-output-functions.sh#L445

In general regarding `Warning` messages see
https://blog.schlomo.schapiro.org/2015/04/warning-is-waste-of-my-time.html

When the subsequent code does not work well with an empty `$opath`
e.g. when
```
cp $v $DISKLAYOUT_FILE /$VM_UID.disklayout.conf
```
is not a supported behaviour, I think it is better for the user
to make a clear decision here and error out e.g. with something like
```
Error "Output path is empty (OUTPUT_URL='$OUTPUT_URL')"
```
The `Error` function shows in which script it was called like
```
# usr/sbin/rear mkrescue
ERROR: Output path is empty (OUTPUT_URL='nfs://192.168.122.1/nfs')
Some latest log messages since the last called script 990_copy_disklayout_file.sh
  ...
```
so the user knows where to look if he was hit by a false error exit
and he needs to adapt the code that causes the false error exit.